catalogue_graph/src/utils/aws.py

```diff
@@ -85,6 +85,19 @@ def get_csv_from_s3(s3_uri: str) -> Generator[Any]:
         yield from csv_reader
 
 
+def write_csv_to_s3(s3_uri: str, items: list[dict]) -> None:
+    if len(items) == 0:
```

Comment:
```suggestion
    if not items:
```
is the more Pythonic idiom.